.github/workflows/update-release-labels.yml

```diff
@@ -0,0 +1,166 @@
+name: Update Release Labels
+
+on:
+  release:
+    types: [published]
```

Comment:
Can we use `workflow_dispatch`? Fixing this job requires backporting a fix into the release branch because this job runs on the commit associated with the release tag.

Reply:
@harupy As we discussed, I supported both release and workflow_dispatch.